.scalafmt.conf

```diff
@@ -0,0 +1,16 @@
+maxColumn = 80
+
+style = IntelliJ
+danglingParentheses = true
+indentOperator = spray
+align.openParenCallSite = false
+align.openParenDefnSite = false
+continuationIndent.callSite = 2
+continuationIndent.defnSite = 2
+optIn.breakChainOnFirstMethodDot = true
+
+spaces {
+  inImportCurlyBraces = false
```

Comment:
Altered this parameter to agree with default IntelliJ behaviour (ow, we need to work out how to change that tool!).